frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewEnvironments/FeatureOverviewEnvironment/EnvironmentAccordionBody/StrategyDraggableItem/StrategyItem/StrategyExecution/hooks/useCustomStrategyParameters.tsx

```diff
@@ -15,7 +15,10 @@ import type {
 import type { IFeatureStrategyPayload } from 'interfaces/strategy';
 
 export const useCustomStrategyParameters = (
-    strategy: CreateFeatureStrategySchema | IFeatureStrategyPayload,
+    strategy: Pick<
+        CreateFeatureStrategySchema | IFeatureStrategyPayload,
+        'parameters' | 'name'
+    >,
```

Comment:
This is the same thing. Lets us use playground strategy results for this. 

Ideally, because this is a new hook, not accept the deprecated `IFeatureStrategyPayload`, but I want to make that refactoring a separate job, because once we start that with new components, it might get a little complex.